ugandaemr_mobile/README.md

```diff
@@ -0,0 +1,8 @@
+# Introduction
```

Comment:
@solemabrothers Add the download, Installation and login content to reduce the number of pages

Reply:
Done

Reply:
@solemabrothers The Introduction has to be a heading 2 so other headings go a level lower